ci/tests.py

```diff
@@ -30,13 +31,18 @@
 
 
 def _availability_zones():
-    api_manager = manager.APIManager()
     try:
+        api_manager = manager.APIManager()
         zones = api_manager.availability_zone_client.list_availability_zones()
         info = zones['availabilityZoneInfo']
         return {zone['zoneName'] for zone in info}
+    except:
```

Comment:
Catch the specific exceptions that you want to catch, and maybe log a message on why did you suppress the exception.